### DISCUSSION TITLE
Renaming MusE

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1222] [Team Posts]
Hi guys,

During the weekend I realized there were a few more linux-audio forums to post the release news to, namely:
https://www.facebook.com/groups/linuxmusicians
https://www.reddit.com/r/linuxaudio/
The reddit forum has more than 10k subscribers so you reach quite a lot of people.

Anyway, what I quickly realized is that several people thought this news were in regards to MuseScore. This caught me a little bit by surprise, but I soon realized it should not. MuseScore is a very popular application and with the recent news about the Muse Group it is clear that this is what people would assume without having more information.

So the question is if we should proactively do something about it, the rebranding Castle suggested came to mind. 
In short, it seems to me that we historically have not been very good at selling the MusE name so I'm personally not averse to trying something new.

What do you guys think about this? Do we have more to lose than to gain?

**Comment by terminator356** [comment]
Bad idea. Rebranding involves changing every piece of text and every variable name with "muse" in it.
I cannot support such a thing. It would kill all my branches-in-waiting.
We are the original MusE. We should officially refer to ourselves as the "MusE Sequencer", not just "MusE".
How much clearer could that be?
Now that those forum users know the truth, there's no point in changing it.
Just promote it - as is. They will follow.

**Comment by kybos** [comment]
> Anyway, what I quickly realized is that several people thought this news were in regards to MuseScore. This caught me a little bit by surprise, but I soon realized it should not. MuseScore is a very popular application and with the recent news about the Muse Group it is clear that this is what people would assume without having more information.

Yes, this is unfortunate but understandable. MusE sounds just like an abbreviation of MuseScore... And the Muse Group now adds to it...

It's obvious that a move to a different and distinctive name would be beneficial in the long term. 
I don't even think that it would be so much effort to implement. Of course the code would have to remain unchanged, except for UI texts containing "MusE" (which should be an exception anyway, as it's usually redundant to repeating the app name inside the app itself).
Similar was done e.g. with XBMC->Kodi rebranding a couple of years ago. The code+repo are still XBMC, the external representation is only Kodi. NB: Kodi is a huge app with lot of docu, references, users etc., still it was possible and worked out well.

I feel "MusE Sequencer" sounds a little outdated and not precise enough, because a sequencer is often associated with MIDI only (but I can be wrong). Apps like MusE are usually called DAW nowadays.
So perhaps "Muse DAW" at least?

What about starting a discussion in the forum, just to get a feeling what the users are thinking about it? We are of course not obliged to comply, and I'm sure there will be different opinions anyway...

**Comment by terminator356** [comment]
> So perhaps "Muse DAW" at least?

Yeah you're right, sequencer doesn't do it justice any more.
MusE DAW would nail it down. No mistaking that one.
It would clearly separate us from the others, and stake our expert 'territory'.
Agreed we could ask the forum'ers, see what they think.

The idea of a completely different name but on the inside it's still MusE feels really slimy to me.
I don't think I could ever get used to that. I mean, which is it - one or the other?
I would be conflicted when naming new variables for example.

**Comment by kybos** [comment]
> The idea of a completely different name but on the inside it's still MusE feels really slimy to me.

Sure it's ugly, it was only meant as a concession to your open branches. On the other side, I assume git should be able to handle variables renaming, so probably it wouldn't be necessary after all.

"Muse DAW" would be more distinctive indeed, but it's still far from optimal. Just look for discussions about the pronunciation of DAW. An example:

"DAW is the worst ever. It’s incredibly cumbersome to pronounce each letter and it’s even worse pronouncing it like a word because it sounds so dumb."
(from https://gearspace.com/board/gear-free-zone-shoot-the-breeze/1185942-pronouncing-quot-daw-quot.html)

That was exactly my feeling before I even started to look it up, but as I'm not a native speaker, I was curious what other people think.

**Comment by spamatica** [comment]
>What about starting a discussion in the forum, just to get a feeling what the users are thinking about it? We are of course not obliged to comply, and I'm sure there will be different opinions anyway...

That might not be a bad idea. Bit of a community building exercise.

**Comment by terminator356** [comment]
OMG LOL I can't believe we're nitpicking about acronyms. WTF?

Perhaps sequencer really is a better choice. After all, it sequences waves as well, no?
That is the purpose, no? To sequence midi and audio?

**Comment by luginf** [comment]
I like "MusE sequencer" as it is. The current name is great too :)
